spec/yaml/swiftnav/sbp/acquisition.yaml

```diff
@@ -70,3 +70,79 @@ definitions:
             desc: |
               PRN-1 identifier of the satellite signal for which
               acquisition was attempted
+
+ - AcqSvProfile:
+    short_desc: Reacq perfomance measurement and debug
```

Comment:
Let's replace 'Reacq' with 'Acq' as this messages can be used for general acquisition performance analysis and debugging.